<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>611</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_appareil">
     <attribute name="title">
      <string>Appareils</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>161</width>
        <height>231</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Ajout</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QPushButton" name="Button_Ajouter">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom de l'appareil</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Propriétaire</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Num</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nomappareil">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_typeappareil">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_proprio">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_numappareil">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>551</width>
        <height>71</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Gestion d'appareils à controler</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>481</width>
         <height>41</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Vous Pouvez ici choisir quels appareils vous voulez controlez ches vos enfants. Ajoutez, modifier et supprimer les appareils selon votre besoin.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>411</width>
        <height>231</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Liste des Appareils</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>391</width>
         <height>151</height>
        </rect>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="cornerButtonEnabled">
        <bool>true</bool>
       </property>
       <property name="rowCount">
        <number>3</number>
       </property>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row>
        <property name="text">
         <string>01</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>02</string>
        </property>
       </row>
       <row/>
       <column>
        <property name="text">
         <string>Appareil</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Type</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Propriétaire</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Num</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>PC de Ramy</string>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string>Ordinateur</string>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string>Ramy</string>
        </property>
       </item>
       <item row="0" column="3">
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string>iPhone de Ramy</string>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string>Mobile</string>
        </property>
       </item>
       <item row="1" column="2">
        <property name="text">
         <string>Ramy</string>
        </property>
       </item>
       <item row="1" column="3">
        <property name="text">
         <string>1</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>190</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>190</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_sites">
     <attribute name="title">
      <string>Proxies</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>645</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
